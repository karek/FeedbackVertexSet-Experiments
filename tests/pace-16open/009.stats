110,364,71
