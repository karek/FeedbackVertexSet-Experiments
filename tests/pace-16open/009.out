21
81 18 29 15 72 20 94 43 1 57 83 8 41 100 106 80 90 88 108 104 91 
